### DISCUSSION TITLE
[Enhancement]: Remove /region/ from the song/album.link url

### DISCUSSION THREAD

**Discussion by brotherelric** [ciderapp/Cider#623] [Feature Request]
### The feature you would like

Please remove the region part from the song.link and album.link url's.
![imagen](https://user-images.githubusercontent.com/66126306/157542527-4a56684a-def5-4dba-a55f-e5a8800af482.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by coredev-uk** [comment]
I did do this originally, but the development team were reporting issues with the links.
